src/pathwayvis/components/map/map.component.ts

```diff
@@ -413,16 +413,28 @@ class MapComponentCtrl {
     if (selected.method.id === 'fva' || selected.method.id === 'pfba-fva') {
 
       // const fvaData = reactionData;
-      const fvaData = _.pickBy(reactionData, (d) => Math.abs((d.upper_bound + d.lower_bound) / 2) > 1e-7);
+      const fvaData = reactionData;
+      for (const i in fvaData) {
+        if (Math.abs((fvaData[i].upper_bound + fvaData[i].lower_bound) / 2) < 1e-7) {
+          delete fvaData[i];
+        }
+      }
+      // const fvaData = _.pickBy(reactionData, (d) => Math.abs((d.upper_bound + d.lower_bound) / 2) > 1e-7);
 
       reactionData = _.mapValues(fvaData, (d) => (d.upper_bound + d.lower_bound) / 2);
 
       this._builder.set_reaction_data(reactionData);
+      console.log("REACTION DATA", reactionData);
       this._builder.set_reaction_fva_data(fvaData);
 
     } else {
       // Remove zero values
-      reactionData = _.pickBy(reactionData, (value: number) => Math.abs(value) > 1e-7);
+      for (const i in reactionData) {
```

Comment:
Same here. I could imagine a pickBy utility function implemented by us in utils.ts